Libraries/Components/Clipboard/Clipboard.js

```diff
@@ -10,24 +10,45 @@
  * @flow
  */
 'use strict';
-
-const Clipboard = require('NativeModules').Clipboard;
+const NativeEventEmitter = require('NativeEventEmitter');
+const RCTClipboard = require('NativeModules').Clipboard;
 
 /**
- * `Clipboard` gives you an interface for setting and getting content from Clipboard on both iOS and Android
+ * `Clipboard` gives you an interface for setting and getting content from 
+ * Clipboard on both iOS and Android. You can subscribe to `clipboardChanged`, 
+ * and `Clipboard` will tell you when the content has changed.
+ * ```javascript
+ * componentDidMount(){
+ *   Clipboard.addListener('clipboardChanged', this._clipboardChanged.bind(this));
+ * }
+ *
+ * componentWillUnmount(){
+ *   Clipboard.removeListener('clipboardChanged', this._clipboardChanged);
+ * }
+ * ```
  */
-module.exports = {
+class Clipboard extends NativeEventEmitter {
+
+  /**
+   * Do not use. The exported module will be a Singleton instance.
+   */
+  constructor() {
+    super(RCTClipboard);
+  }
```

Comment:
Because I want to construct NativeEventEmitter, and export an instance `module.exports = new Clipboard();`